src/view/configuration/configuration.jsx

```diff
@@ -580,9 +578,13 @@ const Configuration = ({ formikProps, initInfo }) => {
                                   pattern: currentPattern
                                 })
                                 .then(newPattern => {
-                                  values.instances[
-                                    index
-                                  ].downloadLinkQualifier = newPattern;
```

Comment:
We shouldn't ever attempt to modify Formik state directly. That's what the `setFieldValue` call on the next line is for (which also ensures that views get re-rendered, etc.).